ridgeback_baxter_description/launch/ridgeback_baxter_world.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<launch>
+
+  <!-- Load the combined ridgeback-baxter URDF's to the param server -->
+  <param name="robot_description"
+         command="$(find xacro)/xacro.py --inorder
+         $(find ridgeback_baxter_description)/urdf/ridgeback_baxter.urdf.xacro gazebo:=true"/>
+
+  <!-- Launch baxter_simulator -->
+  <include file="$(find baxter_gazebo)/launch/baxter_world.launch">
+    <arg name="load_robot_description" value="false"/>
+  </include>
+
+  <!-- Launch ridgeback_simulator -->
+  <!-- TODO -->
+
```

Comment:
The launchfile produces this simulation of Baxter & Ridgeback in Gazebo. Of course, none of the Ridgeback controllers are loaded:
![baxter_ridgeback_gazebo](https://cloud.githubusercontent.com/assets/8646015/11819740/7a5f7c40-a32f-11e5-9b6a-92c076868d57.png)